chore!: update edx-platform image version

fsologureng opened the pull request:
Update to latest edx-platform version, w/o edx-s3-compatibility package installed.

fsologureng committed:
chore!: update edx-platform image version

Update to version w/o edx-s3-compatibility package installed.

tomaspemora reviewed (approved):
👌

tomaspemora commented:
deberíamos aplicar también en:

https://github.com/eol-uchile/edx-openuchile/blob/openuchile-release/koa/Dockerfile
https://github.com/eol-uchile/edx-uabierta/blob/eol-release/koa/Dockerfile
https://github.com/eol-virtuallabx/virtuallabx-edx/blob/vlabx-release/koa/Dockerfile
https://github.com/eol-virtuallabx/norteamericano-edx/blob/vlabx-release/koa/Dockerfile

fsologureng committed:
chore: sorts and standardize requirements

 - sorts packages alfabetically.
 - removes \.git to urls.
 - fixes egg's fragments.
 - adds newline to eof when needed.
